### PR TITLE
fix(chore): to make it work with latest parcel plugin

### PR DESCRIPTION
**Pull request by sendilkumarn**
r? @catsigma

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-import { module } from '../crate/src/lib.rs'
```
Comment (xtuc) [inline js/index.js:1]: Maybe we could use a similar technic that we do on webpack; watching the .rs files oursevles.

@catsigma would you be interested in joing us on a video chat?

Comment (sendilkumarn) [reply js/index.js:1]: I am in

Comment (catsigma) [reply js/index.js:1]: The plugin has been updated to `1.2.4` which enables:
```
import module from '../crate/src/lib.rs'
module.run()
```
or
```
import { run } from '../crate/src/lib.rs'
run()
```
or 
```
import module from '../crate/Cargo.toml'
module.run()
```
or
```
import { run } from '../crate/Cargo.toml'
run()
```

Comment (sendilkumarn) [reply js/index.js:1]: I updated :) Thanks

**Pull request merged**